blocks/gd-block-placeholder/src/Placeholder.svelte

```diff
@@ -12,10 +13,11 @@
 <style>
   .container {
     display: flex;
-    background-color: #cc4343;
-    border-radius: 8px;
-    padding: 8px;
+    background-color: var(--kobber-primitives-color-purple-400);
+    border-radius: var(--kobber-primitives-size-8);
+    padding: var(--kobber-semantic-layout-container-padding-large);
     flex-direction: column;
-    font-size: 18px;
+    font-size: var(--kobber-primitives-size-24);
```

Comment:
Husk: primitivene er enhetsløse, så her får du en ugyldig `font-size: 24;` (tilsvarende også for border-radius).

Reply:
Nei.

Vi har snakket om at kobber bør eksponere størrelser med enheter (px og em), men såvidt jeg vet er ingenting gjort med det ennå.

I SØ gjør vi slike utregninger: `--border-radius: calc((var(--kobber-primitives-border-radius-16) * 1px) + (1em - 16px));`.